Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -74,10 +74,7 @@ final class PledgeCTAContainerView: UIView {
       |> \.spacing .~ Styles.gridHalf(1)
 
     _ = self.pledgeCTAButton
-      |> pledgeCTAButtonStyle(
-        isAccessibilityCategory,
-        amountAndRewardTitleStackViewIsHidden: self.titleAndSubtitleStackView.isHidden
-      )
+      |> pledgeCTAButtonStyle
```

Comment:
We no longer need to know whether it's accessibility category or what state we're in